.pre-commit-config.yaml

```diff
@@ -37,13 +37,13 @@ repos:
     hooks:
       - id: pyproject-fmt
 
-  - repo: local
-    hooks:
-      - id: validate-filenames
-        name: Validate filenames
-        entry: ./scripts/validate_filenames.py
-        language: script
-        pass_filenames: false
+  # - repo: local
```

Comment:
Revert.

Reply:
why am i getting error as 
Python was not found; run without arguments to install from the Microsoft Store, or disable this shortcut from Settings > Manage App Execution Aliases
for the validate filenames checking??

Reply:
What happens when you `run python` in the command prompt?

Reply:
It goes to script mode
I even checked python --version
And it gave like 3.11.4